hw/ip/otp_ctrl/dv/otp_ctrl_sim_cfg.hjson

```diff
@@ -29,7 +29,7 @@
                 "{proj_root}/hw/dv/data/common_sim_cfg.hjson",
                 // Common CIP test lists
                 "{proj_root}/hw/dv/data/tests/csr_tests.hjson",
-                "{proj_root}/hw/dv/data/tests/mem_tests.hjson",
+                // "{proj_root}/hw/dv/data/tests/mem_tests.hjson",
```

Comment:
nit: perhaps add TODO to enable this later

Reply:
oh that is right thanks for the reminder, I forget to add that.